cli/src/pcluster/models/cluster.py

```diff
@@ -539,6 +540,24 @@ def _upload_config(self):
                 e, f"Unable to upload cluster config to the S3 bucket {self.bucket.name} due to exception: {e}"
             )
 
+    def _upload_instance_types_data(self):
+        """Upload source config and save config version."""
```

Comment:
Nitpick: Change the function description to describe the config file you are uploading

Reply:
fixed